File: runtime/src/main/java/com/epam/drill/auto/test/agent/penetration/junit/JUnitPenetration.java
package com.epam.drill.auto.test.agent.penetration.junit;

import com.epam.drill.auto.test.agent.penetration.AnnotationStrategy;

public class JUnitPenetration extends AnnotationStrategy {

    public JUnitPenetration() {
        supportedAnnotations.add("@org.junit.Test");
        supportedAnnotations.add("@org.junit.jupiter.api.Test");
        supportedAnnotations.add("@org.junit.jupiter.params.ParameterizedTest");
    }

    @Override
    public String id() {
        return "junit";
    }
}

File: runtime/src/main/kotlin/com/epam/drill/auto/test/agent/penetration/testing/junit/JUnitPenetration.kt
package com.epam.drill.auto.test.agent.penetration.testing.junit

import com.epam.drill.auto.test.agent.penetration.AnnotationStrategy

class JUnitPenetration : AnnotationStrategy() {
    override val id: String = "junit"

    init {
        supportedAnnotations.add("@org.junit.Test")
        supportedAnnotations.add("@org.junit.jupiter.api.Test")
        supportedAnnotations.add("@org.junit.jupiter.params.ParameterizedTest")
    }
}